features/list/src/main/java/com/techcafe/wantum/list/ListFragment.kt

```diff
@@ -0,0 +1,6 @@
+package com.techcafe.wantum.list
+
+import androidx.fragment.app.Fragment
+
+class ListFragment:Fragment(R.layout.fragment_list) {
```

Comment:
```suggestion
class ListFragment : Fragment(R.layout.fragment_list) {
```

すんごい細かいし、たぶん誰かが書き換えるやつだからそのままでいい気がしているんだけど、一応... :reichel3: